device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -725,7 +735,7 @@ public void setOption(String optionName, Object value) throws IllegalArgumentExc
         else if (value == null)
         {
             // Codes_SRS_DEVICECLIENT_12_026: [The function shall trow IllegalArgumentException if the value is null.]
-            throw new IllegalArgumentException("optionName is null");
+            throw new IllegalArgumentException("optionValue is null");
```

Comment:
should it match the parameter name of "value"?

Reply:
I think it makes sense. the setOption has optionName and value and not optionValue.